### ISSUE TITLE
installation stuck on fedora 37

### ISSUE THREAD

**Issue by sshaaf** [RedHat-EMEA-SSA-Team/hetzner-ocp4#263]
ansible-navigator task that is stuck here for hours. 

```
TASK [openshift-4-cluster : Waiting installation to complete] ******************
FAILED - RETRYING: [host]: Waiting installation to complete (60 retries left).
FAILED - RETRYING: [host]: Waiting installation to complete (59 retries left).
FAILED - RETRYING: [host]: Waiting installation to complete (58 retries left).
FAILED - RETRYING: [host]: Waiting installation to complete (57 retries left).
FAILED - RETRYING: [host]: Waiting installation to complete (56 retries left).
```

And this from the openshift_install.log
.openshift_install.log
```
time="2023-03-30T22:42:19+02:00" level=debug msg="Still waiting for the cluster to initialize: Cluster operator console is not available"
time="2023-03-30T22:44:14+02:00" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console"
time="2023-03-30T22:47:07+02:00" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console"
time="2023-03-30T22:51:56+02:00" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console"
time="2023-03-30T22:56:29+02:00" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.11.12: 783 of 803 done (97% complete)"
time="2023-03-30T22:56:29+02:00" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.11.12: 784 of 803 done (97% complete)"
time="2023-03-30T22:56:29+02:00" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.11.12: 786 of 803 done (97% complete)"
time="2023-03-30T22:56:40+02:00" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.11.12: 786 of 803 done (97% complete)"
time="2023-03-30T22:58:12+02:00" level=debug msg="Still waiting for the cluster to initialize: Cluster operator console is not available"
time="2023-03-30T23:01:33+02:00" level=debug msg="Still waiting for the cluster to initialize: Cluster operator console is not available"
time="2023-03-30T23:05:21+02:00" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console"
time="2023-03-30T23:05:32+02:00" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console"
```

**Comment by rbo** [comment]
I don't know. Never tried on fedora. Is the load balancer running? What is OpenShift saying why the installation is not completed? (`oc get co`)

**Comment by sshaaf** [comment]
I understand adding Fedora is a customization from what is provided. I tried the following.

1. The host file doesn't exist; Created it
ansible/roles/openshift-4-cluster/tasks/prepare-host-Fedora-37.yml

2. loadbalancer didn't come up. add --privileged to it and it worked by editing the following file
ansible/roles/openshift-4-loadbalancer/templates/systemd.service.j2

3. qemu-kvm path was wrong fixed that too
diff --git a/ansible/roles/openshift-4-cluster/templates/vm.xml.j2 b/ansible/roles/openshift-4-cluster/templates/vm.xml.j2
```   
<suspend-to-disk enabled="no"/>
   </pm>
   <devices>
-    <emulator>/usr/libexec/qemu-kvm</emulator>
+    <emulator>/usr/bin/qemu-kvm</emulator>
 {% if vm_storage_backend is defined and vm_storage_backend == "lvm" %}
     <disk type='block' device='disk'>
       <driver name='qemu' type='raw'/>
```

I probably missed some more changes but that's how far I came.

Currently, CentOS stream8 or Rocky; doesn't work out of the box either. And that's what is provided by hetzner by default.
